# Goal
racecar_move_car/Goal mcGoal

---
#Result
int16 mcResult   #-2 is an uninitialized variable

---
#Feedback
int16 mcFeedback   #-2 is an uninitialized variable
